packages/@react-spectrum/s2/src/TagGroup.tsx

```diff
@@ -186,13 +377,58 @@ function TagGroup<T extends object>(
   );
 }
 
-/** Tags allow users to categorize content. They can represent keywords or people, and are grouped to describe an item or a search request. */
-let _TagGroup = /*#__PURE__*/ (forwardRef as forwardRefType)(TagGroup);
-export {_TagGroup as TagGroup};
+function ActionGroup(props) {
+  let {
+    actionsRef,
+    tagState,
+    size,
+    isCollapsed,
+    handlePressCollapse,
+    onAction,
+    actionLabel
+  } = props;
+  let tagListCtx = useContext(TagListContext);
+  // @ts-ignore how do I fix this one?
+  let {id: gridId} = tagListCtx ?? {};
```

Comment:
Looks like `id` is not in the type for `TagListProps`. Not sure if it should be?